Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -24,6 +24,36 @@ span.DropDownMenuSelect
   cursor: pointer;
 }
 
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled)
+{
+  --color-standard-text: var(--color-primary-text);
```

Comment:
please don't.
I don't think we should override the standard properties in a usecase-specific way. This feels like it could have unintended sideeffects in a theoretical future scanario.

in WebButton we just set color, doesn't this work, here, too? Otherwise, we should intoduce a variable and switch that one as needed.